src/WebApp/Views/VehicleManagement/Index.cshtml

```diff
@@ -19,6 +20,14 @@
                 <td>@vehicle.LicenseNumber</td>
                 <td>@vehicle.Brand</td>
                 <td>@vehicle.Type</td>
+                @if (vehicle.HasMaintenanceHistory)
```

Comment:
Nog een `else` statement toevoegen voor het geval er nog geen History is te zien

Reply:
✅